app/helpers/observations_helper.rb

```diff
@@ -33,8 +33,6 @@ def owner_naming_line(obs)
     end
   end
 
-  private
-
   # name portion of Observation title
   def obs_title_consensus_id(name)
```

Comment:
Seems like this should stay private.  I noticed that there are some direct calls from tests, but it's not clear to me why.  It seems like the tests should know exactly what they are expecting.

Reply:
Done. I had disabled the "no ivars in helpers" cop here in `obs_title_consensus_id` awhile back, and am now refactoring the method to take kwargs and re-enabling the cop on the method. (the ivar is already defined by the caller when necessary, so can be easily passed as an arg)